tests/Unit/Services/ParsableContentTest.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+declare(strict_types=1);
+
+use App\Services\ParsableContent;
+use App\Services\ParsableContentProviders\BrProviderParsable;
+use App\Services\ParsableContentProviders\CodeProviderParsable;
+use App\Services\ParsableContentProviders\HashtagProviderParsable;
+use App\Services\ParsableContentProviders\ImageProviderParsable;
+use App\Services\ParsableContentProviders\LinkProviderParsable;
+use App\Services\ParsableContentProviders\MentionProviderParsable;
+use App\Services\ParsableContentProviders\StripProviderParsable;
+
+covers(ParsableContent::class);
```

Comment:
Added to begin adding mutation testing to the codebase, I didn't want to add much more in this PR, because we have a lot of tests needed for full coverage on this class.

<img width="697" alt="Screenshot 2024-09-26 at 12 49 24 AM" src="https://github.com/user-attachments/assets/0cb9a2fa-28bc-4b6a-81c3-52334aaf861d">